src/com/owncloud/android/ui/dialog/LoginWebViewDialog.java

```diff
@@ -122,23 +121,17 @@ public void onAttach(Activity activity) {
                 throw new IllegalStateException("Null authentication method got to onAttach");
             }
             Handler handler = new Handler();
-            switch (authenticationMethod) {
-                case BEARER_TOKEN:
-                    mOAuthWebViewClientListener = (OAuthWebViewClientListener) activity;
-                    mWebViewClient = new OAuthWebViewClient(activity, handler, mOAuthWebViewClientListener);
-                    break;
-                case SAML_WEB_SSO:
-                    mSsoWebViewClientListener = (SsoWebViewClientListener) activity;
-                    mWebViewClient = new SAMLWebViewClient(activity, handler, mSsoWebViewClientListener);
-                    break;
-                default:
+
+            if(SAML_WEB_SSO == authenticationMethod) {
```

Comment:
I guess this part will be deleted when implementing Chrome custom tabs with SAML, have you found any blocker and decided not to take advantage of this PR and get rid of webview in SAML as well?

Reply:
SAML as it is implemented for ownCloud requires WebView. We therefore can't remove Webview.